pincer/objects/guild/guild.py

```diff
@@ -493,6 +497,8 @@ async def modify_member(
         ...
 
     async def modify_member(self, _id: int, reason=None, **kwargs) -> GuildMember:
+        if kwargs.get("communication_disabled_until") is MISSING:
+            kwargs.pop("communication_disabled_until", None)
```

Comment:
I don't think this is required. If it is replace it with
```suggestion
        if kwargs.get("communication_disabled_until") is MISSING:
            kwargs.pop("communication_disabled_until")
```